cohere/client.py

```diff
@@ -269,6 +276,16 @@ def chat(
                 >>>     return_prompt=True)
                 >>> print(res.text)
                 >>> print(res.prompt)
+            Query generation example:
+                >>> res = co.chat(query="What are the tallest penguins?", mode="search_query_generation")
```

Comment:
i think we want to use the enum values for mode, not strings right? this applies to the tests as well

Reply:
that can be somewhat annoying to use though, having to import the enum object from somewhere, and other endpoints just take strings.

Reply:
This should work with both importing mode and with the string